xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/edit/editableProperty.js

```diff
@@ -133,13 +133,20 @@ define('editableProperty', ['jquery', 'xwiki-meta'], function($, xcontext) {
       var editInput = editor.find(':input').filter(':visible');
       // If we cannot find any kind of editInput, then we're in a weird edge case
       // and we don't want to apply any of the following changes.
-      if(!editInput) { return; }
+      if (!editInput) {
+        return;
+      }
       editInput.focus();
-      // Make sure the edit input has an ID, and use the name of the input as a fallback
+      // Make sure the edit input has an ID, and use the name of the input as a fallback.
+      // We add some random UUID to make sure we don't inadvertently collide with another ID.
       if (!editInput.attr('id')) {
-        editInput.attr('id', editInput.attr('name'));
+        let id = editInput.attr('name');
+        if(typeof crypto?.randomUUID === "function") {
+          id = id + '-' + crypto.randomUUID().toString();
```

Comment:
```suggestion
          id += '-' + crypto.randomUUID();
```

The ``toString()`` call seems unnecessary when performing string concatenation.